404.qmd

```diff
@@ -0,0 +1,11 @@
+---
+title: "You’ve entered the garden of missing pages"
+---
+
+Let's get you back to greener grounds.
+
+👉 Go to [homepage](/index.qmd).
+
+![](/_extensions/seedcase-theme/images/404.svg){fig-alt="An illustration of the number 404 surrounded by trees and mountains"}
```

Comment:
Almost. Since the path of the extension in repos that *use* it, rather than develop it, is `seedcase-project/seedcase-theme`, we have to change it here to correct that.

```suggestion
![](/_extensions/seedcase-project/seedcase-theme/images/404.svg){fig-alt="An illustration of the number 404 surrounded by trees and mountains"}
```